src/frontend/api/axios.ts

```diff
@@ -36,6 +36,13 @@ export const håndterApiRessurs = <T>(
                 status: RessursStatus.FEILET,
             };
             break;
+        case RessursStatus.FUNKSJONELL_FEIL:
```

Comment:
Kult! Hva nøyaktig kan en funksjonell feil være f.eks.?

Reply:
Alt fra valideringsfeil (du har ikke gjennomført vilkårsvurderingen) til feil som at SB prøver å gjennomføre et steg man ikke har lov til.

Det hjelper oss ikke så mye frontend, men backend trenger vi det for å ikke logge ting som error (som lager alerts) på ting som egentlig er normal oppførsel.